src/sentry/static/sentry/app/views/settings/components/dataPrivacyRulesPanel/dataPrivacyRulesPanelSelectorField.tsx

```diff
@@ -291,9 +403,9 @@ const StyledTextField = styled(TextField)<{error?: string}>`
     `}
 `;
 
-const SuggestionsWrapper = styled('ul')`
+const SuggestionsDropdown = styled('ul')`
   position: absolute;
-  width: 100%;
+  width: 200%;
```

Comment:
hmm...not sure about it... it looks a bit odd with 200% :/

Reply:
maybe it would be better to move this field under the others with 100%?

Reply:
I changed it back